src/scripts/main.js

```diff
@@ -1 +1,78 @@
 'use strict';
+
+function showNotification(message, isSuccess) {
+  const notification = document.createElement('div');
+
+  notification.setAttribute('data-qa', 'notification');
+
+  notification.className = isSuccess
+    ? 'notification success'
+    : 'notification error';
+  notification.textContent = message;
+
+  document.body.append(notification);
+}
+
+const firstPromise = new Promise((resolve, reject) => {
+  let clicked = false;
+
+  document.addEventListener('click', (e) => {
+    if (e.button === 0) {
+      clicked = true;
+      resolve('First promise was resolved on a left click in the document');
+    }
+  });
+
+  setTimeout(() => {
+    if (!clicked) {
+      reject(
+        new Error('First promise was rejected in 3 seconds if not clicked'),
+      );
+    }
+  }, 3000);
+});
+
+const secondPromise = new Promise((resolve) => {
+  document.addEventListener('click', (e) => {
+    resolve('Second promise was resolved');
+  });
+
+  document.addEventListener('contextmenu', (e) => {
+    resolve('Second promise was resolved');
+  });
```

Comment:
The `secondPromise` resolves on both click and contextmenu events. This means it will resolve twice if both events occur, which might not be the intended behavior. Consider using a flag to ensure it resolves only once.